services/PostService.js

```diff
@@ -0,0 +1,19 @@
+const Post = require('../models/Post');
+
+class PostService {
+  getOnePost(postId) {
+    return Post.find({ _id: postId });
```

Comment:
You should use `findById()` http://mongoosejs.com/docs/api.html#findbyid_findById

Reply:
changed